shared-bindings/audiocore/Mixer.c

```diff
@@ -146,54 +163,6 @@ STATIC mp_obj_t audioio_mixer_obj___exit__(size_t n_args, const mp_obj_t *args)
 }
 STATIC MP_DEFINE_CONST_FUN_OBJ_VAR_BETWEEN(audioio_mixer___exit___obj, 4, 4, audioio_mixer_obj___exit__);
 
-
-//|   .. method:: play(sample, *, voice=0, loop=False)
-//|
-//|     Plays the sample once when loop=False and continuously when loop=True.
-//|     Does not block. Use `playing` to block.
-//|
-//|     Sample must be an `audiocore.WaveFile`, `audiocore.RawSample`, or `audiocore.Mixer`.
-//|
-//|     The sample must match the Mixer's encoding settings given in the constructor.
-//|
-STATIC mp_obj_t audioio_mixer_obj_play(size_t n_args, const mp_obj_t *pos_args, mp_map_t *kw_args) {
-    enum { ARG_sample, ARG_voice, ARG_loop };
-    static const mp_arg_t allowed_args[] = {
-        { MP_QSTR_sample,    MP_ARG_OBJ | MP_ARG_REQUIRED },
-        { MP_QSTR_voice,     MP_ARG_INT | MP_ARG_KW_ONLY, {.u_int = 0} },
-        { MP_QSTR_loop,      MP_ARG_BOOL | MP_ARG_KW_ONLY, {.u_bool = false} },
-    };
-    audioio_mixer_obj_t *self = MP_OBJ_TO_PTR(pos_args[0]);
-    check_for_deinit(self);
-    mp_arg_val_t args[MP_ARRAY_SIZE(allowed_args)];
-    mp_arg_parse_all(n_args - 1, pos_args + 1, kw_args, MP_ARRAY_SIZE(allowed_args), allowed_args, args);
-
-    mp_obj_t sample = args[ARG_sample].u_obj;
-    common_hal_audioio_mixer_play(self, sample, args[ARG_voice].u_int, args[ARG_loop].u_bool);
-
-    return mp_const_none;
-}
-MP_DEFINE_CONST_FUN_OBJ_KW(audioio_mixer_play_obj, 1, audioio_mixer_obj_play);
-
-//|   .. method:: stop_voice(voice=0)
-//|
-//|     Stops playback of the sample on the given voice.
-//|
```

Comment:
Do we want to leave these two functions in for backwards compatibility? They can make the equivalent MixerVoice calls.

Reply:
I can try putting them back in. With the associated space-constraints above, I'll probably approach it with use of `#if CIRCUITPY_AUDIOIO_COMPAT` and `#if CIRCUITPY_AUDIOMIXER` to not further introduce running out of space.

Reply:
This is working within the context of `audiomixer`. But, since moving to a separate module, it will still require updates to old code using `audioio`.

Reply:
> But, since moving to a separate module, it will still require updates to old code using audioio.

Scratch this. `audioio` will still work for boards that include `audiomixer`; see https://github.com/adafruit/circuitpython/pull/2080#discussion_r319523516